Sources/MapboxCoreNavigation/EHorizon/EHorizonDistanceInfo.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import MapboxNavigationNative
+
+/**
+ DistanceInfo is an internal data of OpenLRLineLocation.
+ It's not intended to be read or set from the outside.
+ */
+public struct DistanceInfo {
+
+    public let distance: Double
```

Comment:
Should we use `CLLocationDistance` for such values here and other EH types?

Reply:
`CLLocationDistance` is measured in meters. What unit is this distance measured in? Is it the same unit as `GraphPath.length`?

/cc @SiarheiFedartsou

Reply:
All distances are in meters.